python/controls/window-drag-area/no-frame-window.py

```diff
@@ -1,10 +1,9 @@
-import flet
+import flet as ft
 from flet import Container, IconButton, Page, Row, Text, WindowDragArea, colors, icons
```

Comment:
When using `import flet as ft`, you need to remove the from `flet import x, y, z`
Code looks like [this](https://flet.dev/docs/controls/divider#examples) later on.